2.4/contrib/etc/mongodb.conf

```diff
@@ -0,0 +1,21 @@
+# mongodb.conf
+
+port = 27017
+#fork = true
+pidfilepath = /opt/rh/mongodb24/root/var/run/mongodb/mongodb.pid
+dbpath = /var/lib/mongodb
+
+# Disable data file preallocation.
+noprealloc = true
```

Comment:
since the DB images aren't STI images (coughcoughcough) i'm not sure how we'd enable apps to supply the DB config files.  Maybe some volume mount mechanism but i don't have that full picture in my head.